cmd/sovereignnode/chainSimulator/tests/epochChange/epochChange_test.go

```diff
@@ -55,13 +64,35 @@ func TestSovereignChainSimulator_EpochChange(t *testing.T) {
 
 	defer cs.Close()
 
-	trie := cs.GetNodeHandler(0).GetStateComponents().TriesContainer().Get([]byte(dataRetriever.PeerAccountsUnit.String()))
+	nodeHandler := cs.GetNodeHandler(0)
```

Comment:
Would be better to use `core.SovereignChainShardId` instead of `0`

Reply:
Right, did as suggested